errai-data-binding/src/test/java/org/jboss/errai/databinding/client/test/DataBindingIntegrationTest.java

```diff
@@ -1588,4 +1589,14 @@ public void testDeclarativeBindingWithOnKeyUpForTextInput() throws Exception {
     invokeEventListeners(view.value, "keyup");
     assertEquals(view.value.getValue(), model.getValue());
   }
+
+  @Test
+  public void testTypeInNonBiandablePakageIsNotBound() {
+    try {
```

Comment:
Thank you for adding this test! One thing I've noticed is that I think this test would succeed if type were bindable (i.e. if not exception is thrown).

Here is a pattern I've used in other tests that I think would be useful here:

    try {
      somethingThatShouldHaveAnError();
      fail("Something succeeded but should have failed.");
    } catch (AssertionError ae) {
      throw ae;
    } catch (Exception ex) {
      assertTrue(someCondition(ex));
    }